### PR TITLE
Add more advanced SPA example, and Config per enviorement

### PR DESCRIPTION
**Pull request by amitaibu**
https://github.com/Gizra/generator-elmlang-example/pull/5

### CODE REVIEW

File: generators/app/templates/src/elm/Config.elm [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+prodBackend : Model
+prodBackend =
+    { backendUrl = "https://api.github.com"
+    , name = "gh-pages"
+    , hostname = "example.com"
+    }
+
+
+backends : List Model
```
Comment (amitaibu) [inline generators/app/templates/src/elm/Config.elm]: This should actually be a `Dict`

Comment (amitaibu) [inline generators/app/templates/src/elm/Config.elm]: (I'm going to change it)

**Comment by amitaibu** [comment]
I think this is ready to go.

**Comment by DavidHernandez** [comment]
Reviewed and tested, everything seems to be working fine.

**Comment by amitaibu** [comment]
Great. Can you publish to npm as well.

**Comment by DavidHernandez** [comment]
Published as @1.5.0

**Pull request merged**